class.phpmailer.php

```diff
@@ -857,7 +849,7 @@ protected function addAnAddress($kind, $address, $name = '')
                 return true;
             }
         } else {
-            if (!array_key_exists(strtolower($address), $this->ReplyTo)) {
+            if (!isset($this->ReplyTo[strtolower($address)])) {
```

Comment:
While I know it *usually* works, I'm not keen on using `isset` to check if array elements exist since it breaks if you ever have null elements, so I tend to always use `array_key_exists`.